lib/active_record/tasks/sqlserver_database_tasks.rb

```diff
@@ -70,11 +70,14 @@ def structure_dump(filename, extra_flags)
         dump.gsub!(/^GO\n/, "")                               # Strip db GO statements
         dump.gsub!(/nvarchar\(8000\)/, "nvarchar(4000)")      # Fix nvarchar(8000) column defs
         dump.gsub!(/nvarchar\(-1\)/, "nvarchar(max)")         # Fix nvarchar(-1) column defs
+        dump.gsub!(/varbinary\(-1\)/, "varbinary(max)")       # Fix varbinary(-1) column defs
         dump.gsub!(/text\(\d+\)/, "text")                     # Fix text(16) column defs
+        dump.gsub!(/\r\n/, "\n")                              # Convert line breaks
+        dump.gsub!(/\s+$/, "")                                # Strip trailing whitespace
```

Comment:
I'm using these two.. probably don't need to opt-everyone in to them, but thought I'd share